src/course-outline/data/thunk.js

```diff
@@ -150,3 +150,22 @@ export function updateCourseSectionHighlightsQuery(sectionId, highlights) {
     }
   };
 }
+
+export function publishCourseSectionQuery(sectionId) {
+  return async (dispatch) => {
+    dispatch(updateSavingStatus({ status: RequestStatus.IN_PROGRESS }));
```

Comment:
please change to 
`dispatch(updateSavingStatus({ status: RequestStatus.PENDING }));` for `POST` method

Reply:
Done